packages/presentation/lib/ui/splash_screen/splash_widget.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/base/bloc_screen.dart';
+import 'package:presentation/navigation/base_page.dart';
+import 'package:presentation/ui/splash_screen/bloc/splash_bloc.dart';
+
+const image = 'packages/presentation/images/Frame.png';
+
+class SplashWidget extends StatefulWidget {
```

Comment:
better rename  SplashScreen and other same

Reply:
I call widgets, widgets) it's easier for me to navigate